ledger/metrics/src/main/scala/com/daml/metrics/api/opentelemetry/OpenTelemetryFactory.scala

```diff
@@ -170,6 +179,17 @@ case class OpenTelemetryTimer(name: String, histogram: LongHistogram, timerConte
       context: MetricsContext
   ): Unit = update(duration.toNanos, TimeUnit.NANOSECONDS)
 }
+object OpenTelemetryTimer {
+
+  private[opentelemetry] val TimerUnit: String = "seconds"
+  val TimerUnitAndSuffix: MetricName = MetricName("duration", TimerUnit)
+
+  private val NanoSecondsInASecond = 1_000_000_000
```

Comment:
Nit: For consistency:
```suggestion
  private val NanosecondsInASecond = 1_000_000_000
```

Reply:
Or
```suggestion
  private val NanosecondsPerSecond = 1_000_000_000
```